test/e2e/common/expansion.go

```diff
@@ -144,4 +144,64 @@ var _ = framework.KubeDescribe("Variable Expansion", func() {
 			"test-value",
 		})
 	})
+
+	/*
+		    Testname: var-expansion-subpath
+		    Description: Make sure a container's subpath can be set using an
+			expansion of environment variables.
+	*/
+	framework.ConformanceIt("should allow substituting values in a volume subpath ", func() {
```

Comment:
I don't think this can be part of the conformance suite because it's a new feature

Reply:
I think using `It()` instead of `framework.ConformanceIt()` is sufficient to not make it a conformance test.